modules/gitops-configuring-gitops-managed-resources-to-use-mounted-secrets.adoc

```diff
@@ -0,0 +1,106 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-gitops-managed-resources-to-use-mounted-secrets_{context}"]
+= Configuring {gitops-shortname} managed resources to use mounted secrets
+
+You must configure the {gitops-shortname} managed resources by adding volume mounts configuration to a deployment and configure the container pod to use the mounted secret. For example, consider that you want to add volume mounts configuration to the deployment of `app-taxi` application and the `100-deployment.yaml` file is in the `/environments/dev/apps/app-taxi/services/taxi/base/config/` directory.
+
+.Prerequisites
+
+* You have the AWS Secrets Manager resources stored in your {gitops-shortname} repository.
+* You have the Secrets Store Container Storage Interface (SSCSI) driver configured to mount secrets from AWS Secrets Manager.
+
+.Procedure
+
+. Add volume mounts configuration in the target resource:
+
+.. Add the volume mounting to the deployment YAML file and configure the container pod to use the mounted secret:
++
+.Example deployment file to use the secret provider class resources
```

Comment:
```suggestion
.Example
```

Reply:
@Srivaralakshmi I am suggesting moving the information from the "Example" statement to the text so if your example changes in the future, you don't have to edit the word "Example".

Reply:
I think we should avoid plain "Example" headings and be at least a little descriptive.